modules/op-adding-triggers.adoc

```diff
@@ -161,8 +171,48 @@ Alternatively, you can create the `EvenListener` resource directly from the `pip
 ----
 $ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/03_triggers/04_event_listener.yaml
 ----
-
-. Expose the `EventListener` service as an {product-title} route to make it publicly accessible:
+.. Create a route with the re-encrypt TLS termination:
++
+[source,terminal]
+----
+$ oc create route reencrypt --service=<svc-name> --cert=tls.crt --key=tls.key --ca-cert=ca.crt --hostname=<hostname>
+----
++
+Alternatively, you can create a re-encrypt TLS termination YAML file to create a secured route.
++
+.Example Re-encrypt TLS Termination YAML of the Secured Route
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  name: route-passthrough-secured <1>
+spec:
+  host: www.example.com
```

Comment:
This YAML is supposed to do the same thing as the command above. The command doesn't refer to `www.example.com` so we should unify that. If I understand it correctly, this is the same as `--hostname` above so we could use `<hostname>` here.  Is that correct @praveen4g0 ?

Reply:
@ppitonak Yes i agree with you 
I think @Srivaralakshmi copied content from openshift docs

lets keep in sync so that it won't create confusion

@Srivaralakshmi 
could you change 
`host: www.example.com` to `host: <hostname>`

will that be ok @ppitonak ?

Reply:
yes

Reply:
Done.

Reply:
@ppitonak Hi Pavol, I have incorporated the change suggested. PTAL and kindly provide LGTM. Thanks!